project/AkkaBuild.scala

```diff
@@ -21,6 +17,13 @@ import scala.collection.breakOut
 
 object AkkaBuild {
 
+  object CliOptions {
+    // CI is the env var defined by Github Actions and Travis:
+    // - https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables
+    // - https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
+    val runningOnCi: CliOption[Boolean] = CliOption("akka.ci-server", sys.env.contains("CI"))
```

Comment:
I thought it was used in more places but a quick search says only for the GC-logs, so we could probably remove it.

Reply:
I've declared it because the logic now considers both `akka.ci-server` property and `CI` env var, and I thought that having a `CliOption` makes it slightly clearer. But yeah, I can remove it and check both directly.